tests/test_data.py

```diff
@@ -47,14 +48,15 @@ def __init__(self, length: int, num_classes: int, shape: tuple[int, int]):
             dtype=torch.float32,
             # Move channel dimension to front
         ).permute(2, 0, 1)
+        self._rng = np.random.default_rng(seed=5965)
```

Comment:
Is the specific seed important? If so, should probably add a comment

Reply:
I'll add a comment. In short any seed would work, and the way I pick any seed is with `$RANDOM`.